cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -135,7 +135,7 @@ func (cm *certManagerClient) Images() ([]string, error) {
 	// Gets the cert-manager objects from the embedded assets.
 	objs, err := cm.getManifestObjs()
 	if err != nil {
-		return []string{}, nil
+		return []string{}, nil // nolint:nilerr // If there are no manifests we don't want to error, just return
```

Comment:
I'm not sure if we really want to ignore the err here.

Reply:
Agreed, this seems something to be fixed